<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>954</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>机器人-静电枪控制程序</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>861</width>
      <height>800</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>441</width>
        <height>671</height>
       </rect>
      </property>
      <property name="title">
       <string>ESS-S3011静电枪参数设置</string>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>电压设置：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>接触方式：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>触发方式：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>间隔时间：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>重复次数：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="Discharge_comboBox">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>50</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>直接接触</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>空气接触</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="Trigger_comboBox">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>140</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>本机</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>枪</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>外部</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="voltage_lineEdit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>220</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>0.20</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>230</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>千伏</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="interval_lineEdit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>320</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>330</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>秒</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="repeat_count_lineEdit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>410</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>10</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>420</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>次</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>510</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>已完成次数：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>510</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>次</string>
       </property>
      </widget>
      <widget class="QLabel" name="completed_label">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>510</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="ess_setting_pushButton">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>600</y>
         <width>231</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>参数设置</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="start_pushButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>700</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>311</width>
        <height>661</height>
       </rect>
      </property>
      <property name="title">
       <string>状态显示</string>
      </property>
      <widget class="QLabel" name="tcp_server_connect_status_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>未连接</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>静电枪参数写入状态：</string>
       </property>
      </widget>
      <widget class="QLabel" name="ess_setting_finished_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>未写入</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>机器人连接状态：</string>
       </property>
      </widget>
      <widget class="QLabel" name="limit_day_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>600</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>540</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>使用期限：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>290</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>串口打开状态</string>
       </property>
      </widget>
      <widget class="QLabel" name="com_open_status_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>350</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>未打开</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>410</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>服务器监听状态</string>
       </property>
      </widget>
      <widget class="QLabel" name="tcp_server_listen_status_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>470</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>未监听</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="stop_pushButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>700</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>艾利特机器人连接设置</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>监听端口：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="TcpPort_lineEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>8899</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tcpServer_listen_pushButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>301</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>监听</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>服务器IP：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="tcpServer_ip_comboBox">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>361</width>
        <height>671</height>
       </rect>
      </property>
      <property name="title">
       <string>ESS-S3011静电枪串口设置</string>
      </property>
      <widget class="QComboBox" name="DataBit_comboBox_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>220</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="BaudRate_comboBox_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>139</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comlist_comboBox_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>49</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>320</y>
         <width>101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>校验位：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>波特率：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="ParityBit_comboBox_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>320</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>220</y>
         <width>101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>数据位：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>串口号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>410</y>
         <width>101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>停止位：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="StopBit_comboBox_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>410</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="openCom_pushButton_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>520</y>
         <width>251</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="back_pushButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>700</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>301</width>
        <height>351</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="title">
       <string>状态显示</string>
      </property>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>串口打开状态</string>
       </property>
      </widget>
      <widget class="QLabel" name="com_open_status_label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>未打开</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>服务器监听状态</string>
       </property>
      </widget>
      <widget class="QLabel" name="tcp_server_listen_status_label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>未监听</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="setting_action"/>
    <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="reg_action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="setting_action">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="reg_action">
   <property name="text">
    <string>注册</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
